### ISSUE TITLE
`wrap in *` functions remove escape character `\` in text selection

### ISSUE THREAD

**Issue by papr** [msiniscalchi/atom-latextools#82]
The wrap-in-* functions remove all `\` characters in the selected string they are applied on.

Bug might arise because `getTextInBufferRange()` does not include escape characters  (e.g. [snippet-manager.coffee#L26-L28](https://github.com/msiniscalchi/atom-latextools/blob/master/lib/snippet-manager.coffee#L26-L28)).
```
te = atom.workspace.getActiveTextEditor()
range = te.getSelectedBufferRange()
text = te.getTextInBufferRange(range)
```

**Comment by papr** [comment]
I am using version `0.8.1` and the problem is unfortunately not solved completely yet. When wrapping escaped strings with more than one `\` then the escaping `\`s are all lost but the first one.

Way to reproduce problem:
1. Open a `LaTeX` document.
2. Type and select following string: `\_\_`
3. Execute key combo for wrapping the string (e.g. `cmd-l cmd-m` under OS X for wrapping the selected string in a `texttt` block.)
4. Resulting string is `\__` wrapped in the respective block (depending on the command used in 3.)

**Comment by ig0774** [comment]
@papr: This is fixed in c3a3305, which should be in the next release.
